test/test.expected

```diff
@@ -7,11 +7,11 @@
  (i32.const 0)
 )
 (module
- (type $i32_i32_=>_i32 (func (param i32 i32) (result i32)))
```

Comment:
Just out of curiosity do you know why these are no longer named?

Reply:
They probably just changed the way the printing works.

Reply:
IIRC they actually encourage you to use unique types for everything so it can optimize better, so they probably stopped naming the types like this so there aren't conflicts.